Sources/ProjectDescription/Target.swift

```diff
@@ -92,7 +98,9 @@ public struct Target: Codable, Equatable {
         additionalFiles: [FileElement] = [],
         buildRules: [BuildRule] = [],
         mergedBinaryType: MergedBinaryType = .disabled,
-        mergeable: Bool = false
+        mergeable: Bool = false,
+        odrInitialInstallTags: [String] = [],
+        odrPrefetchOrder: [String] = []
```

Comment:
We probably should combine these two fields into one group like a struct or whatever.